### PR TITLE
Update GraphQL schema

### PR DESCRIPTION
**Pull request by yun-kwak**
Update GraphQL schema, according to the Sketch design.

### CODE REVIEW

File: server/src/database.js [hunk 1]
```diff
@@ -227,12 +227,16 @@ const createStore = () => {
...
-    zip: Sequelize.STRING,
-    street: Sequelize.STRING,
-    additionalAddress: Sequelize.STRING,
+    // TODO(yun-kwak): Split the address into
+    // country: Sequelize.STRING,
+    // state: Sequelize.STRING,
+    // city: Sequelize.STRING,
+    // zip: Sequelize.STRING,
+    // street: Sequelize.STRING,
+    // additionalStreetAddress: Sequelize.STRING,
```
Comment (shberry) [inline server/src/database.js]: Remove the comment block

File: server/src/schema.js [hunk 2]
```diff
@@ -32,6 +33,26 @@ const typeDefs = gql`
...
+    input TagInput {
+        name: String
+    }
+    
+    enum EventSort {
+        BEST_MATCH,
+        NEWEST,
+        MOST_MEMBERS,
+        TIME_CLOSEST,
+
```
Comment (shberry) [inline server/src/schema.js]: remove extra line

File: server/src/database.js [hunk 3]
```diff
@@ -227,12 +227,16 @@ const createStore = () => {
...
     startDateTime: Sequelize.DATE,
     endDateTime: Sequelize.DATE,
-    country: Sequelize.STRING,
-    state: Sequelize.STRING,
-    city: Sequelize.STRING,
-    zip: Sequelize.STRING,
-    street: Sequelize.STRING,
-    additionalAddress: Sequelize.STRING,
+    // TODO(yun-kwak): Split the address into
+    // country: Sequelize.STRING,
```
Comment (ethkim) [inline server/src/database.js]: For any merged code, remove commented code.

File: server/src/schema.js [hunk 4]
```diff
@@ -46,23 +67,28 @@ const typeDefs = gql`
...
-        state: String
-        zip: String
-        additionalAddress: String
+        age: Int # TODO(yun-kwak): Calculate the age of the user from birthday
+        # TODO(yun-kwak): Split the address into
+        # street address
+        # additional street address
+        # city
+        # state
+        # zip code
```
Comment (shberry) [inline server/src/schema.js]: why are these in each line? I don't see the point here

File: server/src/schema.js [hunk 5]
```diff
@@ -71,10 +97,34 @@ const typeDefs = gql`
...
+        maxParticipantNum: Int
+    }
+    
+    type EventBookClub implements Event{
+        id: ID!
+        host: User!
+        thumbnailUrl: String
+        creationTime: DateTime!
+        # When the schedule of the event is updated,
+        # lastUpdatedTime of the event also need to be updated.
```
Comment (shberry) [inline server/src/schema.js]: Why is this sentence separated into two lines?

File: server/src/schema.js [hunk 6]
```diff
@@ -71,10 +97,34 @@ const typeDefs = gql`
...
+        lastUpdatedTime: DateTime!
+        schedule: [EventSchedule]!
+        title: String!
+        description: String!
+        price: Float!
         # TODO(arin-kwak): Implement image uploading feature
+        # image: Upload!
```
Comment (shberry) [inline server/src/schema.js]: remove commented code

File: server/src/schema.js [hunk 7]
```diff
@@ -99,14 +149,16 @@ const typeDefs = gql`
...
-        additionalAddress: String!
-        latitude: String!
-        longtitude: String!
+        # TODO(yun-kwak): Split the address into
+        # country: String!
+        # state: String!
+        # city: String!
+        # zip: String!
+        # street: String!
+        # additionalStreetAddress: String!
```
Comment (shberry) [inline server/src/schema.js]: why are these separated into each line?

**Pull request merged**